pkg/collector/corechecks/cluster/kubernetes_apiserver.go

```diff
@@ -295,7 +294,7 @@ func (k *KubeASCheck) parseComponentStatus(sender aggregator.Sender, componentsS
 				statusCheck = metrics.ServiceCheckCritical
 				message = condition.Error
 			}
-			sender.ServiceCheck(KubeControlPaneCheck, statusCheck, k.KubeAPIServerHostname, tagComp, message)
+			sender.ServiceCheck(KubeControlPaneCheck, statusCheck, "", tagComp, message)
```

Comment:
wondering why we were doing this? maybe the a hostname is required by the backend to consider the event valid?

Reply:
not sure, it seems to have been an oversight that this value was never set. but even with the empty param, we default to associating the hostname that the check was run on with the check. (can see examples on the check summary page)